spring-xd-dirt/src/main/java/org/springframework/xd/dirt/job/dsl/Link.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.xd.dirt.job.dsl;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Represents a node in a {@link Graph} object that Flo will display as a block.
```

Comment:
good catch, thanks for all the feedback Glenn. As I say, I had code blindness and you've picked up on a few things i need to sort out. I've also got some new features already that I might fold in with the next version of the PR (Flo needs a graph-to-DSL conversion function and I have that working now).